### ISSUE TITLE
Blue on blue in Note notices fails color contrast

### ISSUE THREAD

**Issue by neal-timpe** [redhat-documentation/redhat-docs-template#6]
![image](https://github.com/redhat-documentation/redhat-docs-template/assets/12660270/bbe572c7-1edf-46f9-84e5-cd608e7bb197)
![image](https://github.com/redhat-documentation/redhat-docs-template/assets/12660270/6f7f2b7c-a9c1-4fb1-bd1c-4a5d06345a5d)
The link text inside a note text box does not pass color contrast.
